### PR TITLE
Add documentation on how to use Grape on Rails

### PR DESCRIPTION
**Pull request by matsimitsu**
Intercom conversation that sparked this pullreq: https://app.intercom.io/a/apps/yzor8gyw/respond/inbox/14894/conversations/11069755913

### CODE REVIEW

File: source/ruby/integrations/grape.html.md [hunk 1]
```diff
@@ -75,3 +75,30 @@ We have an [example application][example-app] in our examples repository on
...
 
 [example-app]: https://github.com/appsignal/appsignal-examples/tree/grape
+
+
+## Grape on Rails
+
+Grape is often used in combination with Ruby on Rails as a mounted application.
+To make sure performance and error metrics from both Grape and Rails are captured we recommend the following configuration:
+
+1. [Install Appsignal for Rails](/ruby/integrations/rails.html) as described in the documentation
```
Comment (tombruijn) [inline source/ruby/integrations/grape.html.md]: I'd say, use "App_S_ignal"

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.3
+2.3.3
```
Comment (tombruijn) [inline .ruby-version]: problem with the `2.3` selector?

Comment (matsimitsu) [inline .ruby-version]: Rbenv keeps telling me there is no 2.3 :(

Comment (tombruijn) [inline .ruby-version]: ah... rbenv...
I'm using chruby and it automatically selects `2.3.3` (even with 2.3.0 installed)
So I'm 👍  on this, just strange that it's needed.

**Pull request merged**